pulsar-client-api/src/main/java/org/apache/pulsar/client/api/transaction/Transaction.java

```diff
@@ -29,6 +29,53 @@
 @InterfaceStability.Evolving
 public interface Transaction {
 
+    enum State {
+
+        /**
+         * When the transaction is in the `OPEN` state, it can produce with transaction and ack with the transaction.
+         *
+         * When the transaction is in the `OPEN` state, it can commit or abort.
+         */
+        OPEN,
+
+        /**
+         * When the client invokes commit, the state will change to `COMMITTING` from `OPEN`.
```

Comment:
```suggestion
         * When a client invokes a commit, the transaction state is changed from `OPEN` to `COMMITTING`.
```
Do you mean this?

Reply:
Write in the simple present tense as much as possible if you are covering facts that were, are, and forever shall be true. https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.e8uqh1awkcnp